scripts/checks/pragma-validity.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
```

Comment:
Does this script work? I try running it and get various errors.

Reply:
It runs on my end. What error are you getting ?

Reply:
Note that it takes an argument to run it. see the corresponding task in `package.json`